.template.rs

```diff
@@ -0,0 +1,18 @@
+// This file is part of rust-web/twig
+//
+// For the copyright and license information, please view the LICENSE
+// file that was distributed with this source code.
+
+/// Twig library for rust
+///
+/// @author NAME <EMAIL>
+
+/////////////
```

Comment:
Can we do away without these? It's the first time I am seeing such comments in Rust code.